common/locales/en/settings.json

```diff
@@ -39,6 +39,9 @@
     "xml": "(XML)",
     "json": "(JSON)",
     "customDayStart": "Custom Day Start",
+    "changeCustomDayStart": "Change Custom Day Start?",
+    "sureChangeCustomDayStart": "Are you sure you want to change your custom day start?",
+    "nextCron": "WARNING: Your dailies will reset next on <%= time %>.",
```

Comment:
Cron doesn't run exactly at the CDS (unless you're using Habitica at that time), so the wording should reflect this. Maybe something like "Your Dailies will next reset the first time you use Habitica after <%= time %>."